src/main/java/mate/jdbc/dao/CarDao.java

```diff
@@ -1,13 +1,12 @@
 package mate.jdbc.dao;
 
 import java.util.List;
-import java.util.Optional;
 import mate.jdbc.model.Car;
 
 public interface CarDao {
     Car create(Car car);
 
-    Optional<Car> get(Long id);
+    Car get(Long id);
```

Comment:
no, here should be `Optional`, because car with such id may not exist, 
but on the service layer we should handle this situation and throw an exception or return another car or something else